src/PrettyConvert/converters.js

```diff
@@ -70,19 +81,21 @@ export const converters: { [string]: ?Function } = {
   },
   generic: (type: K.Generic, components: Components, depth: number) => {
     if (type.value && type.typeParams) {
-      // As Flow does not know what the keyword Array<T> means, we're doing a check here for generic types with a nominal value of 'Array'
+      // As Flow does not know what the keyword Array<T> means, we're doing a
+      // check here for generic types with a nominal value of 'Array'
       // If a type meets this criteria, we print out its contents as per below.
       return (
         <span>
           <components.TypeMeta>
             {convert(type.value)} <components.Outline>{'<'}</components.Outline>
           </components.TypeMeta>
-          <components.Indent>
```

Comment:
Using indent here was massive overkill, as almost all instances of generic's typeParams are a single argument.